src/components/UserManagement/DuplicateNamePopup.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+
+/**
+ * Modal popup to delete the user profile
+ */
+const DuplicateNamePopup = React.memo(props => {
+  const closePopup = e => {
+    props.onClose();
+  };
+
+  return (
+    <Modal isOpen={props.open} toggle={() => props.popupClose()}>
+      <ModalHeader toggle={() => props.popupClose()}>Duplicate Names</ModalHeader>
+      <ModalBody>
+        <p>There is already a user with the same name.</p>
+        <p>Do you wish to proceed and have duplicate names?</p>
+        <ModalFooter>
+          <Button
+            color="primary"
+            onClick={() => {
+              props.createUserProfile(true);
+              props.onClose();
+            }}
+          >
+            Confirm
+          </Button>
+          <Button onClick={() => props.popupClose()}>Cancel</Button>
```

Comment:
I think we can use "Change name" rather than "Cancel".  It will be more clear.

You can also set focus to name fields in the form. (optional)

Reply:
Good catch Prabhjot! We're allowing the same phone number because fake data is sometimes used for all the testing, and the odds of the same number actually being entered by accident for two people is so low. Some orgs may not care about the number too, and will add ALL fake info... so we don't want that to create problems for orgs like us that use real info.

Reply:
Hey @PrabhjotSembhi, as @one-community said we won't check for duplicate phone numbers. I did update the text of the button from "Cancel" to "Change name".

Reply:
Thank you both